Update docs and its build action (#494)
broadinstitute/gatk-sv: website/docs/modules/index.md

---

The pipeline is written in [Workflow Description Language (WDL)](https://openwdl.org),

> consisting of multiple modules as listed in the following. 

Reviewer: ```suggestion
consisting of multiple modules to be executed in the following order: 
```
